pom.xml

```diff
@@ -106,6 +106,10 @@
       <name>Tomáš Neuberg</name>
       <email>neuberg@m-atelier.cz</email>
     </contributor>
+    <contributor>
```

Comment:
@DevDengChao Please remove this, likely we will drop this entire section at some point as github gives us this data separately already once a contribution comes in.

Reply:
I'm ok after the above is addressed, then this can be merged.

Reply:
@hazendaz Done .